Workbooks/IoT Central/IoT Edge health snapshot/iotedge-health-snapshot.workbook

```diff
@@ -0,0 +1,872 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "4f033854-b3ad-49fb-8e20-83074359af05",
+            "version": "KqlParameterItem/1.0",
+            "name": "resourceType",
+            "type": 7,
+            "isRequired": true,
+            "isGlobal": true,
+            "value": "Microsoft.IoTCentral/IoTApps",
+            "isHiddenWhenLocked": true,
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "jsonData": "[\"Microsoft.IoTCentral/IoTApps\"]"
+          },
+          {
+            "id": "ac81a4e3-288e-4368-b0b1-99f936e9f147",
+            "version": "KqlParameterItem/1.0",
+            "name": "resource",
+            "type": 5,
+            "isRequired": true,
+            "isGlobal": true,
+            "isHiddenWhenLocked": true,
+            "typeSettings": {
+              "additionalResourceOptions": [
+                "value::1"
+              ],
+              "showDefault": false
+            },
+            "defaultValue": "value::1"
+          },
+          {
+            "id": "59a4f202-f0c0-4ca3-9f36-bea664c4f751",
+            "version": "KqlParameterItem/1.0",
+            "name": "invokedFromFleetViewWorkbook",
+            "type": 1,
+            "isRequired": true,
+            "isGlobal": true,
+            "query": "print \"no\"",
+            "isHiddenWhenLocked": true,
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 0,
+            "resourceType": "{resourceType}"
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "{resourceType}"
+      },
+      "conditionalVisibility": {
+        "parameterName": "neverset",
+        "comparison": "isNotEqualTo"
+      },
+      "name": "parameters - 7"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "## This workbook is designed to be invoked from the IoT Edge Fleet view workbook.",
```

Comment:
if this is true, you don't actually need to expose it at all, you can leave it here, and remove this entry from the gallery.  things that are used "inside" other things don't need to be exposed at the top level in the gallery